content/hi/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें."
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डेप्लॉय करते हैं.
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है.
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl ठीक से कॉन्फ़िगर किया गया है:
+
+```shell
+kubectl cluster-info
+```
+
+यदि आप एक URL प्रतिक्रिया देखते हैं, तो kubectl आपके क्लस्टर को एक्सेस करने के लिए सही ढंग से कॉन्फ़िगर किया गया है.
+
+यदि आपको निम्न जैसा संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं किया गया है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है.
+
+```
+The connection to the server <server-name:port> was refused - did you specify the right host or port?
+```
+
+उदाहरण के लिए, यदि आप अपने लैपटॉप (स्थानीय रूप से) पर कुबेरनेट्स क्लस्टर चलाने का इरादा रखते हैं, तो आपको पहले मिनीक्यूब जैसे टूल को इंस्टॉल करना होगा और फिर ऊपर बताए गए कमांड को फिर से चलाना होगा।
```

Comment:
```suggestion
उदाहरण के लिए, यदि आप अपने लैपटॉप (स्थानीय रूप से) पर कुबेरनेट्स क्लस्टर चलाने का इरादा रखते हैं, तो आपको पहले मिनीक्यूब (minikube) जैसे टूल को इंस्टॉल करना होगा और फिर ऊपर बताए गए कमांड को फिर से चलाना होगा।
```

Reply:
try mentioning the English names of certain tools/ technologies as well (discussed above as well)